jaxrs/src/test/scala/com/cerner/beadledom/jaxrs/provider/ForwardedHeaderFilterSpec.scala

```diff
@@ -0,0 +1,266 @@
+package com.cerner.beadledom.jaxrs.provider
+
+import java.net.URI
+
+import javax.ws.rs.container.ContainerRequestContext
+import javax.ws.rs.core._
+import org.mockito.Mockito
+import org.scalatest.{BeforeAndAfter, BeforeAndAfterEach, FunSpec, Matchers}
+import org.scalatest.mockito.MockitoSugar
+
+class ForwardedHeaderFilterSpec extends FunSpec with BeforeAndAfter with Matchers with BeforeAndAfterEach
+  with MockitoSugar {
+
+  def createContainerRequestContext: ContainerRequestContext = {
+    val request = mock[ContainerRequestContext]
+    val uriInfo = mock[UriInfo]
+    Mockito.when(uriInfo.getRequestUriBuilder).thenReturn(UriBuilder.fromUri(new URI("http://hello.there")))
+    Mockito.when(request.getUriInfo).thenReturn(uriInfo)
+    request
+  }
+
+  describe("For a request with just the X-Forwarded-Proto header"){
+    describe("when the header has a value of https"){
+      it("changes the request to https"){
+        val request = createContainerRequestContext
+        val securityContext = mock[SecurityContext]
+        Mockito.when(securityContext.isSecure).thenReturn(false)
```

Comment:
Can this be imported so we don't have to do `Mockito.when`?

Reply:
Yep: https://github.com/cerner/beadledom/pull/149/commits/2b4745132d5a93bb5340b45d98954dd095262209

Reply:
Yep, I imported it and some other `Mockito` methods I used: https://github.com/cerner/beadledom/pull/149/commits/2b4745132d5a93bb5340b45d98954dd095262209